AUDIO/FORCE_RADIO_PLAY_NEXT_STATION.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: AUDIO
+apiset: client
+aliases: ["_0xFF266D1D0EB1195D"]
+---
+## FORCE_RADIO_PLAY_NEXT_STATION
```

Comment:
I ran this name on the FxDK hasher utility and it doesn't match the hash provided I'm getting `0xFF8F0CE9` when it should be `0x53DB6994`.

1. Copy `FORCE_RADIO_PLAY_NEXT_STATION`
2. Open FxDK and head to the hamburger menu on the top left corner of the screen.
3. Click on 'Hasher'.
4. Paste `FORCE_RADIO_PLAY_NEXT_STATION` in there, the hash doesn't match.

The name for this is `SET_RADIO_RETUNE_UP` instead.